### PR TITLE
Included support for biicode.

### PR DESCRIPTION
**Pull request by markspanbroek**
I added support for uploading ccspec to the biicode.com dependency manager.

As a test, I uploaded the current version under my name: https://www.biicode.com/markspanbroek/ccspec
But it would be better if you could upload and maintain it under your own name.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -1,5 +1,10 @@
 cmake_minimum_required(VERSION 2.8.11)
 
+if(BIICODE)
+   include("biicode.cmake")
```
Comment (zhangsu) [inline CMakeLists.txt]: I'm still learning how Biicode works so please bear with me. I'm reading through

http://docs.biicode.com/c++/adapt_library_guide.html#option-1-let-biicode-do-its-job-in-an-isolated-file

It looks like we are using option 1 here (and I like it better), but option 1 doesn't seem to need a [dependencies] section in biicode.conf (option 2 does though). Can we remove the dependencies secton?

Also let's use 2-space indentation for consistency (also in the conf file). I've fixed the indentation at line 9 of this file. Thanks!

**Comment by markspanbroek** [comment]
I'm also new to Biicode, so no worries :)

I fixed the indentation; good catch!

The dependencies are necessary, unfortunately. Biicode is not able to detect the dependencies between the source files correctly for some reason. So if I don't add the dependencies, I'll get linker errors whenever I try to use the ccspec block in a project.

**Comment by zhangsu** [comment]
OK, sounds good. I published it at https://www.biicode.com/zhangsu/ccspec

**Comment by markspanbroek** [comment]
Great! Thanks.

**Pull request merged**